app/views/pages/index.html.erb

```diff
@@ -10,8 +10,19 @@
 
 <script src="javascript/calender/daypilot-all.min.js" type="text/javascript"></script>
 <script type="text/javascript" src="javascript/pages/pages.js"></script>
+<link type="text/css" rel="stylesheet" href="css/calender/custom_theme.css" />
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
```

Comment:
please edit this line and make it as -
`<link type="text/css"  rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css">`